### PR TITLE
create a resume 

### PR DESCRIPTION
**Pull request by Rexben001**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,19 +1,99 @@
...
-</body>
-</html>
\ No newline at end of file
+        <article>
+          <div class="edge"><h3>Work Experience</h3></div>
+          <details>
+            <summary>Andela</summary>
+            <ul>
+              <li><span class="bold">Role:</span>  Software Engineer</li>
+              <li><span class="bold">Duration</span> : From April. 2019 to present</li>
```
Comment (GianBarGian) [inline index.html]: Instead of a `<span>` with a class here you could use the `<strong>` tag here. It's more elegant.

File: less/index.less [hunk 2]
```diff
@@ -1,59 +1,173 @@
...
-	border: 0;
-	font-size: 100%;
-	font: inherit;
-	vertical-align: baseline;
-}
-/* HTML5 display-role reset for older browsers */
-article, aside, details, figcaption, figure, 
-footer, header, hgroup, menu, nav, section {
-	display: block;
+@import "reset.less";
```
Comment (GianBarGian) [inline less/index.less]: Good job with importing an external LESS module. 
As you already might now now, you could want to break your project in even more LESS modules.

File: less/reset.less [hunk 3]
```diff
@@ -0,0 +1,49 @@
+// Is this working?
```
Comment (GianBarGian) [inline less/reset.less]: Don't forget to clean up your code and cut off all the pointless comments.

File: less/index.less [hunk 4]
```diff
@@ -1,59 +1,173 @@
...
+@article-color: darken(#f2f3f7, 10%);
+@anchor-color: darken(#f2f3f7, 50%);
+@mobileScreen: 500px;
+
+
+.flex-display(@justifyContent) {
+  display: flex;
+  justify-content: @justifyContent;
+  flex-wrap: wrap;
 }
```
Comment (GianBarGian) [inline less/index.less]: Good job at using mixins here!

**Comment by GianBarGian** [comment]
Good job with the project Benjamin, it's really well packed togheter.

Your code looks clean, and your website is looking really good. Well done!

File: index.html (hunk 1 above)
Comment (Rexben001) [reply index.html]: That's true. Thanks

File: less/index.less (hunk 4 above)
Comment (Rexben001) [reply less/index.less]: Thanks

File: less/reset.less (hunk 3 above)
Comment (Rexben001) [reply less/reset.less]: I'd do that

**Pull request merged**